{{cookiecutter.repostory_name}}/app/src/{{cookiecutter.django_project_name}}/settings.py

```diff
@@ -199,13 +241,14 @@
 }
 
 # Sentry
-if env('SENTRY_DSN', default=''):
-    sentry_logging = LoggingIntegration(
-        level=logging.INFO,  # Capture info and above as breadcrumbs
-        event_level=logging.ERROR  # Send error events from log messages
-    )
-
+if SENTRY_DSN := env('SENTRY_DSN', default=''):
     sentry_sdk.init(
-        dsn=env('SENTRY_DSN', default=''),
-        integrations=[DjangoIntegration(), sentry_logging]
+        dsn=SENTRY_DSN,
+        integrations=[
```

Comment:
https://docs.sentry.io/platforms/python/guides/celery/ ?
If sentry is enabled by default, maybe would be good to solve #42?

Reply:
Added redis & celery integrations